docs/schemas/itemRead.json

```diff
@@ -42,10 +42,6 @@
         "callback": {
           "$ref": "itemMeta.json#/definitions/callback"
         },
-        "id": {
```

Comment:
I'm not sure on this. Is the field being populated by anything, if so removing it is going to break things...
@ryango-aotal your thoughts?

Reply:
I am sure it won't break anything. This "id" was added by Brandon.... He added it in the wrong place... lol

Reply:
Here is his commit
https://github.com/talentappstore/tas-tenant-apis/commit/897148e3fbb97c2744c1b8f4e9f22234e8dc5fa3

Reply:
the issue would be that if the field is being populated, it would indicate that there is some java code expecting the itemRead/Write pojos to have a property `id`. Any project which is expecting that pojo property will fail to rebuild.

Reply:
It might be the case, but I cannot think of any project using this id. Brandon added this id for populating pickListChoice id (here is another commit 5e1d2a5), I believe he added this id in the wrong place.
If we really don't want to touch this, I can leave it for now.

Reply:
ok, I'm concerned that we recently added the `elementId` property when there was an existing `id` property. Now we are removing the `id` property which will break java apps. Either way if `elementId` or `id` are removed, code will break.

Reply:
okay, no worries, I can leave this for now but as you can see, the description here ("a unique id for picklist choice") is totally wrong under this pickListItem.

Reply:
The implementing class/sub items as of the moment has the IDs and for consistency, clients should use that.

No one should be using ItemWrite/Read directly.